### PR TITLE
#215 Created Return Button that Appears at the end of each lab

### LINKED ISSUE #215: Create a "Return to Home" button after Quiz
# Description

We should consider adding a reusable "Return to Home" button after the user completes the quiz section.

# Acceptance Criteria

- [x] Reusable return to home button component created
- [x] Return to home button works for every lab
- [x] Return to home button returns to the home page
- [x] Nothing breaks!

### PR DESCRIPTION
**Pull request by dm9965**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

**Review by Mster115 — CHANGES_REQUESTED** [review]
BLOCKED: Until Lab 7 merge to account for hidden certificate check, this would entail if hiddenCertificate then this button is not shown

**Comment by Mster115** [comment]
@dm9965 lab 7 merged

File: client/src/assets/stylesheets/pages/quiz.scss [hunk 1]
```diff
@@ -184,6 +184,12 @@
     padding: 8px !important;
   }
 
+  .homeButton {
+    margin: 40px;
+    padding:15px;
+    background-color: #3f51b5;
+  }
```
Comment (heathermoses) [inline client/src/assets/stylesheets/pages/quiz.scss:191]: Would it be possible to make the button fit to the text like the next and previous buttons? Right now the width seems pretty big.

Screenshot 2023-04-03 at 9 47 03 AM

**Review by heathermoses — CHANGES_REQUESTED** [review]

File: client/src/assets/stylesheets/pages/quiz.scss (hunk 1 above)
Comment (dm9965) [reply client/src/assets/stylesheets/pages/quiz.scss:191]: I'll fix it this weekend!

Comment (dm9965) [reply client/src/assets/stylesheets/pages/quiz.scss:191]: It was fixed this weekend! 🤪

File: client/src/components/quiz/App.js [hunk 2]
```diff
@@ -95,7 +94,8 @@ const QuizHandler = (props) => {
    */
   function onComplete() {
     scoreResults();
-    setQuizCompleted(true);
+    props.setQuizCompleted(true);
+    console.warn(props.quizCompleted);
```
Comment (heathermoses) [inline client/src/components/quiz/App.js]: Nit: I think this console warn can be removed

**Review by heathermoses — APPROVED** [review]
Tested on all labs, works for me!

**Pull request merged**